client/app/telecom/telephony/service/fax/convertToVoicefax/telecom-telephony-service-fax-convertToVoicefax.html

```diff
@@ -39,49 +41,52 @@ <h2 data-translate="loading"></h2>
-                                    id="service"
-                                    class="form-control"
-                                    required
-                                    autofocus
-                                    data-flat-select
-                                    data-ng-model="$ctrl.convertToVoicefaxForm.serviceName"
-                                    data-ng-options="service as service.service for service in $ctrl.services"
-                                    data-ng-change="$ctrl.changeNumberToConvert(service)">
+                                <select name="service"
+                                        id="service"
+                                        class="form-control"
+                                        required
+                                        autofocus
+                                        data-ng-model="$ctrl.convertToVoicefaxForm.serviceName"
+                                        data-ng-options="service as service.service for service in $ctrl.services"
+                                        data-ng-change="$ctrl.changeNumberToConvert(service)">
                                 </select>
                             </div>
                             <div class="form-group">
                                 <div data-contracts="$ctrl.convertToVoicefaxForm.serviceName.informations.contracts"
                                      data-contracts-validated="$ctrl.convertToVoicefaxForm.contracts">
                                 </div>
                             </div>
-                            <div class="form-group bottom-space-m12" data-ng-if="!$ctrl.convertToVoicefaxForm.prices">
-                                <button
-                                    type="submit"
-                                    class="btn btn-primary"
-                                    data-ng-disabled="!$ctrl.convertToVoicefaxForm.contracts || $ctrl.convertToVoicefaxForm.isOrdering">
-                                    <span class="ovh-font ovh-font-1x6 ovh-font-cart right-space-m4" aria-hidden="true"></span>
+                            <div class="form-group mb-3"
+                                 data-ng-if="!$ctrl.convertToVoicefaxForm.prices">
+                                <button type="submit"
+                                        class="btn btn-primary"
+                                        data-ng-disabled="!$ctrl.convertToVoicefaxForm.contracts || $ctrl.convertToVoicefaxForm.isOrdering">
+                                    <span class="ovh-font ovh-font-1x6 ovh-font-cart mr-1" aria-hidden="true"></span>
                                     <span data-translate="telephony_service_fax_convert_to_voicefax_list_service_order"></span>
-                                    <spinner class="left-space-m4" data-ng-if="$ctrl.convertToVoicefaxForm.isOrdering"></spinner>
+                                    <bouncing-box-loader class="d-inline-block align-middle ml-2"
```

Comment:
It could be better if we replace `ovh-font-cart` icon with the `bouncing-box-loader` when `ctrl.convertToVoicefaxForm.isOrdering` is true right?